### ISSUE TITLE
Script Usage with Multiple GoDaddy API Keys

### ISSUE THREAD

**Issue by SoulSeekkor** [acmesh-official/acme.sh#2908]
First, sorry for creating an issue for this as I couldn't find any other way to ask this question.  Second, thank you for this amazing script!  I just set this up and it rocks!  I have multiple wildcards in one certificate set up and it should install those certs and reload nginx automatically when it renews.

Now I have two questions in hopes of completely automating what I have running:

1) Is there a way to automate this command as part of the renew process?
acme.sh  --toPkcs  -d <domain>  [--password pfx-password]

I need the pfx for other processes and this would be a slick inclusion, if nothing else I could cron it of course.

2) Is there a way to use multiple GoDaddy API keys?  I have essentially two GoDaddy accounts I need to create 1 cert for each account (wildcards), but the GD key seems to be account wide where I need it to be cert specific, is this possible?

**Comment by Neilpang** [comment]
@SoulSeekkor 

yes, there are ways to support multiple Godaddy API keys,  but it's not easy enough.

For example,  

1. acme.sh is user account-based, so you can create 2 linux users to install and use acme.sh
2. If just want to use only one account,  you can specify different `--config-home`  or `--accountconf`
https://github.com/acmesh-official/acme.sh/wiki/How-to-install#4-advanced-installation
3. The easiest way I can think of is using [dns alias mode](https://github.com/acmesh-official/acme.sh/wiki/DNS-alias-mode), you can set one domain alias to the other one, so you can use one Godaddy API key to generat certs for both of the domains.

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.
